ee/tabby-webserver/src/schema/analytic.rs

```diff
@@ -32,29 +36,63 @@ pub enum Language {
     Other,
 }
 
+lazy_static! {
+    static ref LANGUAGE_STRING_MAPPINGS: HashMap<Language, Vec<&'static str>> = {
+        let mut map = HashMap::new();
+        for language in Language::iter() {
+            let value = match language {
```

Comment:
Extract into function and use `Language::iter().map(|l| (l.clone(), l.language_names())).collect()`